src/store/components/AdminPreparePage/index.tsx

```diff
@@ -46,17 +46,19 @@ const AdminPreparePage: React.FC<AdminPreparePageProps> = (props) => {
   }
 
   const merchData: Record<string, { quantity: number; name: string; variantType: string; variantValue: string }> = {};
-  pickupEvent.orders?.forEach((order) => {
-    order.items.forEach((item) => {
-      merchData[`${item.option.uuid}`] = merchData[`${item.option.uuid}`] ?? {
-        quantity: 0,
-        name: item.option.item.itemName,
-        variantType: item.option.metadata?.type,
-        variantValue: item.option.metadata?.value,
-      };
-      merchData[`${item.option.uuid}`].quantity += 1;
+  pickupEvent.orders
+    ?.filter((order) => order.status !== OrderStatus.CANCELLED)
+    .forEach((order) => {
+      order.items.forEach((item) => {
+        merchData[`${item.option.uuid}`] = merchData[`${item.option.uuid}`] ?? {
```

Comment:
Does this need a template string? I believe `merchData[item.option.uuid]` would be the same

Reply:
👍